NuGet.config

```diff
@@ -2,20 +2,12 @@
 <configuration>
   <!-- Don't use any higher level config files.
        Our builds need to be isolated from user/machine state -->
-  <fallbackPackageFolders>
-    <clear />
-  </fallbackPackageFolders>
```

Comment:
Any reason to clear this and `disabledPackageSources` below?
I don't think these should be removed, as it may upset the compliance tools.